client/webpack.config.js

```diff
@@ -99,6 +100,8 @@ const config = Object.assign(baseConfig, {
       hot: !!devOrTestMode,
     }),
     new ManifestPlugin({ publicPath: output.publicPath, writeToFileEmit: true }),
+    // only load moment.js data for locales we support (see config/locale.rb)
+    new webpack.ContextReplacementPlugin(/moment[\/\\]locale$/, /en|es|de|it|nb|nl|pt-BR|sv|vi|fr/),
```

Comment:
According to Codeclimate, it looks like you don't need that escape character!

Reply:
It's gone now.  Thanks!